### DISCUSSION TITLE
Nice Introduction!

### DISCUSSION THREAD

**Discussion by Pattiecodes** [skills/.github#340] [Introduction to GitHub]
With this Introduction to GitHub course, I've been shown a lot of basic GitHub functions, and I wish to learn more.

**Comment by SlimBloodworth** [comment]
Hi @Pattiecodes 👋. I'm pretty experienced in git and GitHub. If you're interested, I'd be happy to show you more about GitHub, as well as git.

**Reply by Pattiecodes** [reply]
Hi! Thank you! I'd appreciate any help I can get! 😊
